src/include/cursor.h

```diff
@@ -49,6 +49,9 @@
             F_SET(cursor, WT_CURSTD_EVICT_REPOSITION);                              \
     } while (0)
 
+/* Maximum number of incremental backups. */
+#define WT_BLKINCR_MAX 2
```

Comment:
I'm still confused as to why this is where it is, why is it now in cursor.h?

Reply:
This is because it's as part of "backup" and "backup" is an implementation of a "cursor".

We have to move it somewhere because of dependencies and this looks like an appropriate place for this one.

Reply:
Okay thank you